src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments/Infrastructure/OuterApi/OuterApiService.cs

```diff
@@ -101,5 +101,10 @@ public async Task<GetPriorLearningSummaryQueryResult> GetPriorLearningSummary(lo
         {
             return await _outerApiClient.Get<GetPriorLearningSummaryQueryResult>(new GetPriorLearningSummaryQueryRequest(providerId, cohortId, draftApprenticeshipId));
         }
+
+        public async Task<GetCohortDetailsResponse> GetCohortDetails(long providerId, long cohortId)
```

Comment:
I can't see any references to this method. The mapper uses the request/response objects and the underlying client directly. (Personally I see no point to this entire class, since it adds so little, but that is an aside).

Reply:
Good shout there, will remove.